async_fuse/src/fuse_request.rs

```diff
@@ -725,67 +646,81 @@ impl<'a> Request<'a> {
 #[cfg(test)]
 mod test {
     use super::*;
-    use log::debug;
+
+    #[repr(align(8))]
```

Comment:
why `repr(align(8))`? is there any special consideration?

Reply:
`FuseInHeader` is aligned to 8 bytes. `Align8` is enough for structs used here.
`[u8;N]` is aligned to 1 byte.  Requests which are not well-aligned will cause an alignment error (potential UB).
So we have runtime checks in `ByteSlice::fetch` and `ByteSlice::fetch_all_as_slice`.

Described in <https://github.com/datenlord/datenlord/pull/84/commits/bb4ea0b224b3ccbccb1645ca1dbe1a3dbd24ed87>